misc/__init__.py

```diff
@@ -1 +0,0 @@
-from .util import maybe_decode, xrun, atexitrun
```

Comment:
@winder - all the original [tests still passed](https://app.circleci.com/pipelines/github/algorand/indexer/522/workflows/0c8fe51b-6b45-40e5-9190-82dab856f011/jobs/948) even after removing these imports, but frankly, I'm not totally sure what the impact is. I needed to remove this imports from the `make indexer-v-algod-swagger` command to work (i.e. `pytest -sv misc/parity`)